.npmignore

```diff
@@ -0,0 +1,2 @@
+dist/types
+tsconfig.tsbuildinfo
```

Comment:
why do we need a .npmignore? I find using the `files` property in the package.json better

Reply:
I prefer that as well. This is partly an artifact of a previous design pattern where some js files in types were obsolete, but I refactored.

My perspective is - `files` should be preferred when possible. The caveat is that it doesn't allow exclude patterns. So in cases like ours, this leaves us with the option of manually specifying _every_ file we want to include. 

Having done that I can say that that particular route inevitably causes problems down the road. Whenever we add source files, rename, restructure, etc, we also have to remember to update our whitelist. This often gets forgotten and ends up with a broken build. IDE Refactoring tools also tend not to support it. Using an `.npmignore` is a fine option when you need to add specific excludes in an otherwise included folder.

The articles about this titled things like "For the love of GOD WHY are you doing ....?!?!" make me chuckle. Tech "influencers" are a bizarre bunch, and this has been going on for decades. I remember when you were the devil and a garbage coder if you used a ternary. (Seriously!)

That said, having refactored things, we could get away with changing the `files` field to use `dist/*.js`. I would actually argue against this, however, because generally speaking, this is a case where if it's in the dist folder, we want to package it, unless _explicitly_ defined otherwise. For this purpose, we include the dist dir, globally, and utilize the blacklist as needed.

If you disagree or feel strongly about it, I'd be open to change it, though! Just wanted to make the reasoning clear.

Reply:
Seems I can't use both `files` and a blacklist! 😞 At this point, we only need to exclude one file, so I can do that another way. Dropping .npmignore